### ISSUE TITLE
Chrome Ubuntu loads voices only after page refresh when tts is used on the very first page

### ISSUE THREAD

**Issue by jankapunkt** [jankapunkt/easy-speech#1]
Close all chrome windows, and try to get voices, no voices will be loaded, `onvoiceschanged` never dispatches
Refresh the page -> voices load, `onvoiceschanged` will be dispatched.

**Comment by Zireael07** [comment]
In my case it says no voices even after reload.

**Comment by jankapunkt** [comment]
@Zireael07 when you close -> then open your Chrome and open the developer console (F12) and (maybe repeatedly)  enter `speechSynthesis.getVoices()` then do you get any results?

**Comment by Zireael07** [comment]
no, still says no voices. Tried googling but honestly there is nothing on WHY there are no voices and how to check if your OS has voices

**Comment by jankapunkt** [comment]
Yeah, this is a huge problem in general. Most likely there are no voices installed on your system. You can check if things are running on OS level by this topic: https://askubuntu.com/questions/589074/what-is-speech-dispatcher
